### PR TITLE
Only run CTS issue workflow when opening PRs; run on base branch

### PR DESCRIPTION
**Pull request by psalz**
[Turns out](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request) the `pull_request` trigger by default also runs when PRs are updated or re-opened. Change to only run on the `opened` activity.

Additionally, I forgot that using the `pull_request` trigger doesn't allow the workflow to access repository secrets when the triggering PR originates from a fork. Thankfully there is a `pull_request_target` trigger to remedy this, which runs the workflow in the context of the base branch (i.e., changes to the workflow itself aren't executed). This is fine for our use case, as we don't need to access the PR's contents within the workflow.

~~(Side note: I marked this as `editorial` to not run the workflow - maybe we need another label for these kinds of infrastructure changes).~~ Edit: I've now limited the workflow to only run when changes are made inside the `adoc` directory.

### CODE REVIEW

**Review by gmlueck — APPROVED** [review]

**Comment by gmlueck** [comment]
I marked this "editorial" because it does not change the specification.  Therefore, I think it could be merged without WG approval.

@psalz: I'll wait to hear from you before I merge this.  Let me know when you think it is ready.

**Comment by psalz** [comment]
> I marked this "editorial" because it does not change the specification. Therefore, I think it could be merged without WG approval.
> 
> @psalz: I'll wait to hear from you before I merge this. Let me know when you think it is ready.

Should be good to go!

**Comment by gmlueck** [comment]
Merging as editorial

**Pull request merged**
